packages/components/psammead-story-promo/src/index.jsx

```diff
@@ -230,6 +234,9 @@ const TextGridItem = styled.div`
   }
 `;
 
+/*
+ *  Link
+ */
```

Comment:
This file is definitely getting a little bit too big, so thank you for adding these ⭐️